### DISCUSSION TITLE
日本流行的「友情婚」是什麼？不限性別、是一段「剔除性愛關係」的普通婚姻！

### DISCUSSION THREAD

**Discussion by Jerry10413** [1101-GS4719-C/Python-Community-GS4719-C#1001] [General]
剛剛看到的，我覺得這是一個很新穎的想法，像我以前高中的時候就會很想要有一個人可以共度每一刻，但是在大學後才發現其實有時候自己一個人反而比較好，但是自己一個人的問題就會很多，也沒有辦法處理(尤其是長輩的部分阿哈哈)，但是戀愛這種有時效性很有可能兩方得愛不對等的狀況讓我真的很累不太想再去戀愛，所以我對這篇文章很感興趣
各位有興趣的人也可以去瀏覽看看
https://www.msn.com/zh-tw/lifestyle/other/%E6%97%A5%E6%9C%AC%E6%B5%81%E8%A1%8C%E7%9A%84-%E5%8F%8B%E6%83%85%E5%A9%9A-%E6%98%AF%E4%BB%80%E9%BA%BC-%E4%B8%8D%E9%99%90%E6%80%A7%E5%88%A5-%E6%98%AF%E4%B8%80%E6%AE%B5-%E5%89%94%E9%99%A4%E6%80%A7%E6%84%9B%E9%97%9C%E4%BF%82-%E7%9A%84%E6%99%AE%E9%80%9A%E5%A9%9A%E5%A7%BB/ar-AA15Ve9q?ocid=msedgntphdr&cvid=3d23830f12544a66bc02c8dab7dc09d3

**Comment by delia07152** [comment]
怎麼聽起來就是室友的關係啊哈哈哈，不過沒有法律束縛的關係是說斷就可以斷的，感覺會讓這段關係變得相對於婚姻來的脆弱， 而且這個概念還蠻新穎的，對於要怎麼跟長輩們交代，也是個難題。然後我看文章說只有1%的人嚮往友情婚，感覺好低喔哈哈哈。

**Comment by Tuoresa** [comment]
這樣的概念感覺與心靈伴侶非常相似，但特別用一個「友情婚」的名詞將它畫入類婚姻的界線有點刻意的感覺。雖然生命著實缺少不了他人陪伴，能遇見靈魂伴侶並長時間相處在一起，無關乎性別和愛情真的是一件再幸運不過的事，因此還是很喜歡文章中靈魂契合等想法的。但文章中說到的友情婚的益處感覺也有些屬於自我觀念的調適，並非將兩人關係冠上友情婚就能獲得如此令人嚮往之關係。
不知道把友情婚視為一種心態而非特別約定好的生活模式，不知道會不會更自然一些。

**Comment by Sylvie0601** [comment]
我覺得這對我來說也會非常嚮往，因為是對彼此不用負責的關係，但卻有人能陪伴身旁的感覺，這種自由卻總像婚姻一樣陪伴在身旁度過餘生似乎也是一種不錯的選擇!

**Comment by bonny123865** [comment]
這種感覺就是隨時都能陪伴的朋友吧?!
我也同意樓上有人說不應該把這樣的關係界定成一種模式比較好
感覺就是生活上特別match，相處都很愉快就好，不用特別以這樣一個詞來綁定關係

**Comment by dabbie109303501** [comment]
認同樓上認為的室友
兩個人一起住 不管性別(在此提到是因為不是每個國家都可以同性結婚) 沒有拘束很自由 也沒有性愛關係 那不就跟室友一樣嗎 
那這樣問什麼還要特地去結婚並改變戶口本
如果之後其中一方找到真愛 那兩人離婚還要處理財產問題 不是更麻煩嗎
不是很理解友情婚的意義

**Comment by wenchi9023** [comment]
各取所需我覺得沒有甚麼問題，但一旦雙方有一人打破其中一項共識
那豈不是更麻煩了嗎...我覺得不用特別制定一個形式來綁定對方，感情本來就來來去去，如果硬要綁住對方，只會帶來反效果而已

**Comment by Twenty0522** [comment]
這個想法非常新穎，會想讓人試試看，我覺得這應該比較適合兩個人都非常獨立且需要自己空間的人。

**Comment by Jasper525** [comment]
感覺很多同志在出櫃前結婚（與異性）也是差不多的概念，儘管對自己伴侶沒有性趣，但還是有愛。

**Comment by yuxinlin93** [comment]
感覺很不錯也，就是陪再一起一輩子的朋友，沒有太多壓力。現在雖然有很好的朋友，但未來各自有各自的家庭，能不能那麼常聚在一起也很難說，隨時間漸行漸遠也說不定。這種婚姻對想要自己一個人的人來說蠻友善的，一般時間都有各自的空間，但在需要一個人陪伴的時候，會知道那裏會有一個人永遠在那裏

**Comment by woohsin** [comment]
這篇文章的內容確實觸動到了你內心的一些感受，講述了一種新穎的想法，即一種可以陪伴人共度每一刻的新型關係。這種想法在現代社會中可能會引起人們的共鳴，因為隨著社會變遷和生活方式的改變，人們對於關係的期待也在不斷調整。

對於一個人而言，孤獨和獨處的時刻確實是很寶貴的，但同時，也會帶來一些問題和挑戰，特別是在與家人或社會大眾的互動中。這種新型關係或許可以填補一些缺口，給予人們一種溫暖和陪伴，同時又不需要承擔太多的情感壓力和負擔。

然而，戀愛和人際關係始終是一個復雜的議題，每個人對於愛和依賴的需求都不同，也都有自己的處理方式。無論如何，重要的是要找到適合自己的生活方式和關係模式，並以開放的心態去面對和接納生活中的種種可能性。

**Comment by titan92601** [comment]
文章討論了友情婚姻現象，提出了不同於傳統戀愛觀的選擇。這種新觀念可能為那些對婚姻失去信心的人帶來新希望。然而，我認為婚姻不僅是伴侶關係，還牽涉到家庭、社會等層面。重要的是，尊重每個人的選擇，並確保建立在理解、尊重和溝通的基礎上。

**Comment by NcuJerryKao** [comment]
剛剛看到的這篇文章介紹了日本流行的「友情婚」，這是一個非常新穎的概念。對於那些在高中時期渴望有個人陪伴，但在大學後發現獨處也不錯的人來說，這種婚姻模式提供了一個值得探討的選擇。友情婚是一種不以性別為界限的婚姻關係，重點是彼此之間的友誼，而非浪漫或性愛。

這種關係的好處在於，兩個人可以互相支持，共同面對生活中的挑戰，尤其是在需要處理長輩問題或其他困難時，有個人可以依賴。友情婚可以提供穩定的伴侶關係，而不需要承受傳統戀愛中的情感壓力和不對等的愛情困境，讓雙方在心理和情感上都能感到輕鬆。

對於那些對傳統戀愛感到疲憊的人，友情婚可以是一種解脫。它消除了戀愛關係中的很多不確定因素，讓人們能夠專注於建立穩定和真誠的友誼。這種婚姻形式可以為人們提供一種新的生活方式選擇，使他們能夠享受伴侶關係的好處，而不需要承受戀愛的壓力。

如果你對這種概念感興趣，可以進一步了解這種模式的細節，看看它是否適合你的生活方式和需求。或許在現代社會中，友情婚這種方式可以幫助更多的人找到情感上的歸屬感，而不用再陷入傳統戀愛的困境中。這篇文章提供了一個新的視角，讓我們思考婚姻和伴侶關係的多樣性和可能性。

各位有興趣的人也可以去瀏覽這篇文章，了解更多關於友情婚的資訊和其背後的理念。希望這樣的新思路能給大家帶來啟發，找到最適合自己的生活方式。

**Comment by LinChloe** [comment]
感覺是很棒的想法。我也是那種喜歡一個人的類型，覺得有另一半是一種綑綁的感覺難以掙脫。所以真要說的話比較喜歡和其他人以【朋友】的方式相處。而且自己發現對異性真的沒有感覺到吸引力。但也不想跟同性在一起。難道我是無性戀嗎?

**Comment by hbh10601** [comment]
對於友情婚的想法我保持尊重但我自己不會去施行的態度，對我來說結婚一定要有很強大的感情基礎才能，這樣看起來真的比較像是其他同學所說的室友關係而已。

**Comment by Shang621** [comment]
日本流行的「友情婚」是一種特殊的婚姻形式，不限性別，主要強調建立在深厚友情基礎上的伴侶關係。這種婚姻忽略了性愛關係，僅聚焦於相互支持、理解和共享生活的友誼。這樣的婚姻強調伴侶之間的情感連結和互助關係，而非傳統意義上的浪漫愛情。友情婚在日本社會中愈發普遍，反映了對於婚姻形式的多元化和人們對於伴侶關係的不同期待。這種形式的婚姻可能為那些不追求傳統愛情的人提供了另一種選擇，並將婚姻定義為更為靈活和開放的關係模式。

**Comment by lilishh** [comment]
如果只是需要可以互相支持、理解彼此的人和自己共同生活的話，一定要有婚姻關係或是伴侶關係嗎?我認為能夠這樣一起生活的人，是朋友也無妨，只要能找到很合拍的朋友，一起生活會更加有趣吧，僅僅是朋友的關係也不用對彼此的其他事情負責。

**Comment by sandyho0307** [comment]
我覺得友情婚之所以吸引人，是因為它打破了傳統婚姻中的一些束縛和壓力。很多人在戀愛和婚姻中都會面臨愛情不對等的問題，這會讓雙方都感到疲憊和壓力。

**Comment by asd99263fg** [comment]
如果事情無法以正向思考面對，可以先允許自己接受和面對負面情緒，這是正常的過程。嘗試與信任的朋友或家人分享你的感受，尋求支持和建議。將問題分解為可管理的小步驟，逐步解決，並尋找事情中的學習經驗或成長機會。適度進行運動、冥想或從事讓你放鬆的活動，這有助於減輕壓力並提升情緒。若情況持續無法改善，考慮尋求專業心理諮詢或輔導，獲得專業幫助和支持。
